cluster/images/hyperkube/master.json

```diff
@@ -12,11 +12,12 @@
               "/hyperkube",
               "controller-manager",
               "--master=127.0.0.1:8080",
-              "--min-resync-period=3m",
               "--service-account-private-key-file=/srv/kubernetes/server.key",
               "--root-ca-file=/srv/kubernetes/ca.crt",
+              "--terminated-pod-gc-threshold=100",
```

Comment:
OK, just copied it from `master-multi.json`
I suppose I have to remove it from that file also